test/index.js

```diff
@@ -441,13 +441,13 @@ function fix() {
  * Argument Parsing
  */
 
-function parseArg() {
-  var argv = process.argv.slice(2),
-      options = {},
+function parseArg(argv) {
+  var options = {},
       opt = '',
       orphans = [],
       arg;
 
+  argv = Array.isArray(argv) ? argv.slice(2) : [];
```

Comment:
I'm not sure this check should be there. If `argv` is not an array it should throw an error.